v3.0/tidb-in-kubernetes/deploy/gcp-gke.md

```diff
@@ -52,37 +61,41 @@ The terraform script expects three environment variables. You can let Terraform
     - It is recommended to create a new service account to be used by Terraform. See [this page](https://cloud.google.com/iam/docs/creating-managing-service-accounts) to create a service account and grant `Project Editor` role to it.
     - See [this page](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) to create service account keys, and choose `JSON` key type during creation. The downloaded `JSON` file that contains the private key is the credentials file you need.
 * `TF_VAR_GCP_REGION`: The region to create the resources in, for example: `us-west1`.
-* `TF_VAR_GCP_PROJECT`: The name of the GCP project.
-
-> **Note:**
->
-> The service account must have sufficient permissions to create resources in the project. The `Project Editor` primitive will accomplish this.
+* `TF_VAR_GCP_CREDENTIALS_PATH`: Path to a valid GCP credentials file.
+    - It is recommended to create a new service account to be used by Terraform as shown in the below example.
 
-To set the three environment variables, for example, you can enter the following commands in your terminal and replace the values with the path to the JSON file you have downloaded, the GCP region and your GCP project name:
+Below we will set these environment variables
```

Comment:
Please limit the use of "we" in documentation.
```suggestion
Set these environment variables as follows:
```

Reply:
Jacob is already working off of this branch (PR will be coming as soon as this is merged), so we can make improvements to his PR.